README.md

```diff
@@ -747,7 +817,7 @@ You need `Docker` installed locally to use our `Makefile` workflow.
 * Fork it.
 * Create your feature branch (`git checkout -b my-new-feature`).
 * Install dependencies: `make build`.
-* Fix bugs or add features. Make sure the changes pass the coding guidelines by runing PHP CodeSniffer: `make cs`. You can also run `make cbf` to fix some errors automatically.
+* Install Composer vendor dependencies with `make composer install`
```

Comment:
I removed the entry for coding guidelines (the command does not exist in Makefile). Also added a command to populate vendor dependencies otherwise it throws this error and I'm unable to run unit tests

<details>
<summary>Error when running unit tests</summary>

<img width="1234" alt="Screenshot 2023-04-10 at 6 58 08 PM" src="https://user-images.githubusercontent.com/8046636/230885729-6ae0d1e3-ac02-423a-a646-f1a35115f672.png">

</details>